### ISSUE TITLE
thirdPartyAudit build task spewes gazilion warnings

### ISSUE THREAD

**Issue by bleskes** [elastic/elasticsearch#17436]
I'm not sure if this is an issue with CI, an issue with `thirdPartyAudit` logic or an actual problem, but we should look at it... 

https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+periodic/270/consoleFull

```
[thirdPartyAudit] WARNING: The referenced class 'org.apache.commons.logging.Log' cannot be loaded. Please fix the classpath!
[thirdPartyAudit] WARNING: The referenced class 'org.apache.commons.logging.Log' cannot be loaded. Please fix the classpath!
[thirdPartyAudit] WARNING: The referenced class 'org.apache.commons.logging.Log' cannot be loaded. Please fix the classpath!
[thirdPartyAudit] WARNING: The referenced class 'org.jboss.marshalling.ByteInput' cannot be loaded. Please fix the classpath!
[thirdPartyAudit] WARNING: The referenced class 'org.jboss.marshalling.ByteInput' cannot be loaded. Please fix the classpath!
[thirdPartyAudit] WARNING: The referenced class 'org.jboss.marshalling.ByteInput' cannot be loaded. Please fix the classpath!
[thirdPartyAudit] WARNING: The referenced class 'org.jboss.marshalling.ByteInput' cannot be loaded. Please fix the classpath!
[thirdPartyAudit] WARNING: The referenced class 'org.jboss.marshalling.ByteInput' cannot be loaded. Please fix the classpath!
```

**Comment by nik9000** [comment]
Is this a new thing or has it been doing this in CI the whole time? We interact with the code that does the third party audit by looking at the logger. Usually we capture the output, filter it, and interpret it.

**Comment by bleskes** [comment]
> Is this a new thing or has it been doing this in CI the whole time?

No idea. I just noticed it now. Maybe other people can chime in..

**Comment by javanna** [comment]
Closed by #21443
